pkg/xen-tools/patches-4.14.0/13-qemu-Revamp-mmap-allocation-routine.patch

```diff
@@ -0,0 +1,124 @@
+From 3ad5ce46a00385fc59adb5ba9f7c4b358e346bbf Mon Sep 17 00:00:00 2001
```

Comment:
What is exact root cause that requires us to have this patch?

Reply:
This one is merged from a separate PR, should reduce noise in the memory profilers at least.